### ISSUE TITLE
Spyder wont open after update using pip to 3.0.2

### ISSUE THREAD

**Issue by jcbalmer** [spyder-ide/spyder#3762]
Updated from Spyder 3.0.0 to 3.0.2 on windows 7 using pip:

python -m pip install -U spyder

Update seemed to be successful, but when I then tried to open Spyder I have this error message flash for half a second:

C:\Program Files\WinPython-64bit-3.4.3.6\python-3.4.3.amd64\python.exe: Error while finding spec for 'spyderlib.start_app' <<class 'ImportError'>: No module named 'spyderlib'>

This error message flashes in a window like the command prompt window, the black box with white text. It disappears so fast the only way I could view it was to hit the print screen key the moment after I try to open it and paste the image in word.

## Versions and main components

* Spyder Version: 3.0.2
* Python Version: 3.4.3
* Operating system: WIndows 7

## Dependencies

Please go to the menu entry `Help > Optional Dependencies` (or
`Help > Dependencies`), press the button `Copy to clipboard`
and paste the contents below:

Sorry I am not super computer savvy so not sure what the dependencies section is asking me to do. I have searched google for my error without any luck.

Cheers

**Comment by jcbalmer** [comment]
I tried running 'python -m pip install -U spyder' again in the command prompt but it just says 'Requirement already up-to-date for everything.

**Comment by ccordoba12** [comment]
This seems a problem with WinPython. Any clues @stonebig?

**Comment by stonebig** [comment]
old winpython versions were launching spyder via its former spyderlib directory.
unless copying spyder icon and associated script from a newer winpython, you can't fix it.

or create a "scripts\spyder.bat" bathc with inside:
````
@echo off
call "%~dp0env.bat"
"%WINPYDIR%\scripts\spyder.exe" %*
````
and use it in place of unfixable icon

**Comment by jcbalmer** [comment]
Thank you SOOOO much ccordoba12 and stonebig, that solve the issue a treat! 

For anyone else that has the issue, based on stonebigs recommendations I went into:

C:\Program Files\WinPython-64bit-3.4.3.6\scripts to find a file called spyder.bat

I right clicked on it, clicked edit and passed:

@echo off
call "%~dp0env.bat"
"%WINPYDIR%\scripts\spyder.exe" %*

As stonebig said. Then double clicking spyder.bat opened spyder.

Finally I created a shortcut of spyder.bat on my desktop.
